src/xcode/ENA/ENA/Source/Services/Risk/Provider/Helper/CachedAppConfiguration.swift

```diff
@@ -31,9 +31,10 @@ final class CachedAppConfiguration: AppConfigurationProviding {
 	// MARK: Protocol AppConfigurationProviding
 
 	func appConfiguration(forceFetch: Bool = false) -> AnyPublisher<SAP_Internal_V2_ApplicationConfigurationIOS, Never> {
+		let isOnline = InternetConnectivityMonitor.shared.isDeviceOnline
```

Comment:
aren't we doing a shared class now as discussed?

Reply:
InternetConnectivityMonitor is the shared class which I got from Felix after merged his reopened PR.